app/api/views.py

```diff
@@ -13,5 +13,15 @@ def process_coords():
     Returns: This returns a post request of create_segments().
     """
 
-    data = json.loads(request.form.get("coords")).get("b")
-    return "{}".format(create_segments(data))
+    data = json.loads(request.form.get("coords"))
+    hsegs = []
+    polygons = []
+    for polygon in data:
+    	segment = polygon.get("b")
+        polygons.append(segment);
+    	hsegs.append(create_segments(segment))
+
+    session["polygons"] = polygons
+    session["regions"] = hsegs
+    
+    return "{}".format(session["polygons"])
```

Comment:
Returns `polygons`. The more I think about it. This doesn't need to actually return data. Just a response for success.
```python
    return "", 200
```